deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -65,6 +86,31 @@ exports.handler = async (event, context) => {
    */
   const request = event.Records[0].cf.request;
   const host = request.headers.host[0].value.toLowerCase();
+
+  let { url, status } = resolveFundamental(request.uri);
+  if (url) {
+    return redirect(url, {
+      permanent: status === 301,
+      cacheControlSeconds: 3600 * 24 * 30,
+    });
+  }
+
+  // Starting with /docs/ or empty path (/) should redirect to a locale.
+  // Also trim a trailing slash to avoid a double redirect.
+  if (
+    request.uri.startsWith("/docs/") ||
+    request.uri === "/" ||
+    request.uri === ""
+  ) {
+    const path = request.uri.endsWith("/")
+      ? request.uri.slice(0, -1)
+      : request.uri;
+    const language = guessLanguage(request);
```

Comment:
Should that be 
```suggestion
    const language = guessLanguage(request) || 'en-US';
```
or, perhaps
```suggestion
const { DEFAULT_LOCALE } = require("@mdn/global-constants");

    const language = guessLanguage(request) || DEFAULT_LOCALE;
```

Reply:
indeed